packages/ember-glimmer/tests/integration/helpers/log-test.js

```diff
@@ -0,0 +1,60 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { get } from 'ember-metal/property_get';
+import Logger from 'ember-metal/logger';
+
+var originalLog, logCalls;
+
+moduleFor('Helpers test: {{log}}', class extends RenderingTest {
+
+  constructor() {
+    super();
+
+    originalLog = Logger.log;
```

Comment:
nit: now that we have a class, we can just put it on the instance (`this.originalLogs` and `this.logCalls`)